### ISSUE TITLE
Version 4.02 causes error 125 on New-NavContainer

### ISSUE THREAD

**Issue by mheijkant** [Microsoft/navcontainerhelper#251]
When using version 4.0.2. this causes an 125 error on running New-NavContainer. When going back to version 4.0.1. the script runs without error. The error is as follows:

[DBG]: PS C:\Users\michel\Foodware\PowerShell\Scripts>
Docker version 18.03.1-ee-2, build ebbcd7e
Starting New-NAVContainer: MIN-FIRSTEXT
Host is Windows Server 10.0.14393.0 - ltsc2016
Using image bcinsider.azurecr.io/bcsandbox:base
Creating Nav container MIN-FIRSTEXT
Using license file c:\users\michel\foodware\Licenses\5164953 NAV365 on Premises.flf
NAV Version: 13.1.25341.0-W1
Generic Tag: 0.0.7.0
Container OS Version: 10.0.14393.2551 (ltsc2016)
Host OS Version: 10.0.14393.0 (ltsc2016)
Creating container MIN-FIRSTEXT from image bcinsider.azurecr.io/bcsandbox:base
ff0bb9a93ea72fb18c4c496688940f9c1a7eff0f0dfc4b083bf19f4935ff3b9f
ExitCode: 125
Commandline: docker run --env ClientServicesPort=7821 --name MIN-FIRSTEXT --hostname MIN-FIRSTEXT --env auth=Windows --env username="admin" --env ExitOnError=N --env locale=en-US --env licenseFile="c:\run\my\license.flf" --env databaseServer="" --env databaseInstance="" --volume "C:\ProgramData\NavContainerHelper:C:\ProgramData\NavContainerHelper" --volume "C:\ProgramData\NavContainerHelper\Extensions\MIN-FIRSTEXT\my:C:\Run\my" --restart always --env enableApiServices=Y --env useSSL=N --volume "C:\ProgramData\NavContainerHelper\Extensions\MIN-FIRSTEXT\Program Files:C:\navpfiles" --volume "c:\windows\system32\drivers\etc:C:\driversetc" --env securePassword=76492d1116743f0423413b16050a5345MgB8AEkAaQBVADMAMgB6AHoAeQBZAHoAWQB0AHcAVwB6AEkALwAvAFcAagBYAGcAPQA9AHwANgAyAGMAYwA0ADAAYwA4AGUAMwAyADgANgBlADcAOABjADUAMQA1ADgAZAA1ADYAYwBhADUAZABhADAAYQBjADUANwAzADIAMQA1AGIAMABlADEAYQAzADcAZQAwADMAOABiADQAYwA4ADYAZAAyAGUANQBmADUAMQBmAGYAYQA= --env passwordKeyFile="c:\run\my\aes.key" --env removePasswordKeyFile=Y -v c:\users\michel\foodware\Objects:c:\original -v C:\Users\michel\Schouw\PS\Export\Objects:c:\export -v C:\Users\michel\Schouw\PS\Backup:c:/backup -v C:\Users\michel:c:\compile -e locale=nl-NL -p 7820:7045 -p 7826:80 -p 7821:7821 -p 7822:7047 -p 7824:7049 -p 7827:8080 --security-opt credentialspec=file://MIN-FIRSTEXT.json --ip=172.18.240.125 -p 7825:1433 --env accept_eula=Y --detach bcinsider.azurecr.io/bcsandbox:base
At C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.4.0.2\ContainerHandling\New-NavContainer.ps1:657 char:19
+ ...       if (!(DockerDo -accept_eula -accept_outdated:$accept_outdated - ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,DockerDo

**Comment by kine** [comment]
I had similar problem but at the end it was my fault. My script stopped working because I had nested array/hashtable in the additional parameters. I needed to change how I am passing my additional parameters to the script. What I see in your command is, that there is 
`-v c:\users\michel\foodware\Objects:c:\original -v C:\Users\michel\Schouw\PS\Export\Objects:c:\export -v C:\Users\michel\Schouw\PS\Backup:c:/backup -v C:\Users\michel:c:\compile `
without using double quotes around the paths and in the backup you have forward slash instead backslash.
